t/t6026-merge-attr.sh

```diff
@@ -180,4 +180,28 @@ test_expect_success 'up-to-date merge without common ancestor' '
 	)
 '
 
+cat >./custom-merge <<\EOF
+#$SHELL_PATH
+sleep 1 &
+EOF
+chmod +x ./custom-merge
+
+test_expect_success 'custom merge backend with delay' '
```

Comment:
The label "custom merge backend with delay" describes nicely what this is. Just reading the (short) code would tell us the same, though.

A much more interesting description (one that is currently missing) would be: "custom merge does not lock index".

And that latter description would also help the casual reader to figure out why the heck this needs to be tested.